lib/oli_web/components/delivery/user_account.ex

```diff
@@ -141,6 +142,7 @@ defmodule OliWeb.Components.Delivery.UserAccount do
   attr(:is_admin, :boolean, required: true)
   attr(:class, :string, default: "")
   attr(:dropdown_class, :string, default: "")
+  attr(:show_support_link, :boolean, default: false)
```

Comment:
Added some logic to show a support menu item in the account menu when one is not available from the sidebar nav so that a support button is always available